diff --git a/node_modules/ajv/dist/types/json-schema.d.ts b/node_modules/ajv/dist/types/json-schema.d.ts
index 78ec842..b1cc078 100644
--- a/node_modules/ajv/dist/types/json-schema.d.ts
+++ b/node_modules/ajv/dist/types/json-schema.d.ts
@@ -31,6 +31,8 @@ declare type UncheckedJSONSchemaType<T, IsPartial extends boolean> = (// these t
     type: JSONType<"string", IsPartial>;
 } & StringKeywords : T extends boolean ? {
     type: JSONType<"boolean", IsPartial>;
+} & StringKeywords : T extends Date ? {
+    type: JSONType<"string", IsPartial>;
 } : T extends readonly [any, ...any[]] ? {
     type: JSONType<"array", IsPartial>;
     items: {
