diff --git a/node_modules/@sodaru/yup-to-json-schema/dist/converters/string.js b/node_modules/@sodaru/yup-to-json-schema/dist/converters/string.js
index 7ca2824..29d8bb1 100644
--- a/node_modules/@sodaru/yup-to-json-schema/dist/converters/string.js
+++ b/node_modules/@sodaru/yup-to-json-schema/dist/converters/string.js
@@ -1,6 +1,5 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-var uuidRegExPattern = "/^(?:[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}|00000000-0000-0000-0000-000000000000)$/i";
 var stringConverter = function (string) {
     var jsonSchema = {};
     string.tests.forEach(function (test) {
@@ -31,7 +30,7 @@ var stringConverter = function (string) {
                 jsonSchema.format = "uri";
                 break;
             case "uuid":
-                jsonSchema.pattern = uuidRegExPattern;
+                jsonSchema.format = 'uuid'
                 break;
         }
     });
